raiden-ts/src/raiden.ts

```diff
@@ -309,6 +242,92 @@ export class Raiden {
     );
   }
 
+  /**
+   * Transforms the redux channel state to [[RaidenChannels]]
+   *
+   * @param state - current state
+   * @returns raiden channels
+   */
+  private mapTokenToPartner = (state: RaidenState): RaidenChannels =>
+    transform(
+      // transform state.channels to token-partner-raidenChannel map
+      state.channels,
+      (result: RaidenChannels, partnerChannelMap, tokenNetwork) => {
+        const token = findKey(state.tokens, tn => tn === tokenNetwork) as Address | undefined;
+        if (!token) return; // shouldn't happen, token mapping is always bi-directional
+        result[token] = this.mapPartnerToChannel(partnerChannelMap, token, tokenNetwork);
+      },
+    );
+
+  /**
+   * Returns an object that maps partner addresses to their [[RaidenChannel]].
+   *
+   * @param partnerChannelMap - an object that maps partnerAddress to a channel
+   * @param token - a token address
+   * @param tokenNetwork - a token network
+   * @returns raiden channel
+   */
+  private mapPartnerToChannel = (
+    partnerChannelMap: {
+      [partner: string]: Channel;
+    },
+    token: Address,
+    tokenNetwork: string,
+  ): { [partner: string]: RaidenChannel } =>
+    transform(
+      // transform Channel to RaidenChannel, with more info
+      partnerChannelMap,
+      (partner2raidenChannel, channel, partner) => {
+        const {
+          ownDeposit,
+          partnerDeposit,
+          ownBalance: balance,
+          ownCapacity: capacity,
+        } = channelAmounts(channel);
+
+        partner2raidenChannel[partner] = {
+          state: channel.state,
+          ...pick(channel, ['id', 'settleTimeout', 'openBlock', 'closeBlock']),
+          token,
+          tokenNetwork: tokenNetwork as Address,
+          partner: partner as Address,
+          ownDeposit,
+          partnerDeposit,
+          balance,
+          capacity,
+        };
+      },
+    );
+
+  /**
+   * Initializes the [[transfers$]] observable
+   *
+   * @param state$ - Observable of the current RaidenState
+   * @returns observable of sent and completed Raiden transfers
+   */
+  private initTransfersObservable = (
```

Comment:
Could these functions become `static` methods instead of private member functions? For consistency, the member functions are more useful when we need it to be a member: dynamic definition, memoization, etc.
Or maybe even move them to relevant utils or so (like `raidenSentTransfer`)

Reply:
Do we need it to be static though, as it would expose it also to end users/developers and would be available as `Raiden.initTransfersObservable()` but it seems pretty useful for the outside world. Other than that I'm fine with it 👍

Reply:
I think `static private` is a thing, but not sure. Don't take it hard anyway, just nitpicking, can be just private as is. Thank you

Reply:
I have moved them instead to `src/helper.ts` and `src/state.ts`, as suggested in #717